tests/webview/integration/test_rex_redirects.py

```diff
@@ -90,36 +76,6 @@ def test_minimal_view_for_android_apps(webview_base_url, rex_base_url):
     assert rex_base_url not in response.url
 
 
-@markers.rex
-@markers.nondestructive
-@markers.xfail  # until we resolve and deploy https://github.com/openstax/cnx/issues/357
-def test_cnx_sitemap_exclusion(rex_base_url, archive_base_url):
```

Comment:
Why are we removing this? This is a test set to xfail b/c it hasn't been implemented yet. When we do this work we'll want to check that the books that are on rex are not on the sitemap

Reply:
I removed it due to the comment from Bey in the PR: 'Closing this card because the work no longer needs to happen due to other SEO work being completed.'